src/main/java/rex/util/Parser.java

```diff
@@ -16,6 +16,8 @@
  * It includes methods to parse commands, tasks, and dates, as well as handle input validation.

```

Comment:
Should the javadocs comment start with indicative mood ie "Includes" rather than "It", as the it is already inferred to be the class?

Reply:
Thanks for this suggestion, I believe the current comment should be okay but I will take it into consideration :)